packages/amplify-graphql-auth-transformer/src/vtl-generator/rds/resolvers/subscription.ts

```diff
@@ -0,0 +1,37 @@
+import { and, compoundExpression, ifElse, iff, list, methodCall, not, obj, printBlock, ref, set } from 'graphql-mapping-template';
+import { ConfiguredAuthProviders, RoleDefinition } from '../../../utils';
+import { generateAuthRulesFromRoles, validateAuthResult } from './common';
+
+export const generateAuthExpressionForSubscriptions = (providers: ConfiguredAuthProviders, roles: Array<RoleDefinition>): string => {
+  const expressions = [];
```

Comment:
`providers` input is not used in this method, check and remove.

Reply:
Same as above. These got added here to implement the VTL Generator interface.